### PR TITLE
2.17 announcement blog post

### PR DESCRIPTION
**Pull request by travisbeckham**
Do not merge until Dec 5, 2024 6am Pacific

https://deploy-preview-1890--linkerdio.netlify.app/2024/12/05/announcing-linkerd-2.17/

### CODE REVIEW

File: linkerd.io/content/blog/2024/1205-announcing-linkerd-2.17/index.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+thousands of clusters. Federated services join features such as
+[flat network / pod-to-pod multicluster](/2/tasks/pod-to-pod-multicluster/)
+(introduced in Linkerd 2.14) in the toolbox for this new class of Kubernetes
+adoption.
+
+## Linkerd Day at Kubecon London
+
+Next April we're happy to report that
+[the CNCF is hosting Linkerd Day](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/linkerd-day/)
+at Kubecon London. Many of the Linkerd maintainers will be in attendance, and
```
Comment (kflynn) [inline linkerd.io/content/blog/2024/1205-announcing-linkerd-2.17/index.md:138]:
```suggestion
We're delighted to report that
[the CNCF is hosting Linkerd Day](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/linkerd-day/)
at Kubecon London next April! Many of the Linkerd maintainers will be in attendance, and
```

**Review by kflynn — APPROVED** [review]
Looks great! I suggested one rewording for consideration but I'm approving as-is. 🙂

File: linkerd.io/content/blog/2024/1205-announcing-linkerd-2.17/index.md (hunk 1 above)
Comment (travisbeckham) [reply linkerd.io/content/blog/2024/1205-announcing-linkerd-2.17/index.md:138]: This sounds better to me.

File: linkerd.io/content/blog/2024/1205-announcing-linkerd-2.17/index.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+we're expecting a great lineup of Linkerd talks as well as plenty of Linkerd
+users. Come see us in London!
+
+## Distributions of Linkerd 2.17
+
+[Buoyant](https://buoyant.io/), the creators of Linkerd, has
+[released Buoyant Enterprise for Linkerd 2.17.0](https://buoyant.io/blog/announcing-linkerd-2-17/), and
+published a
+[Linkerd 2.17 changelog](https://docs.buoyant.io/release-notes/buoyant-enterprise-linkerd/enterprise-2.17.0/)
+with additional guidance and content.
```
Comment (wmorgan) [inline linkerd.io/content/blog/2024/1205-announcing-linkerd-2.17/index.md:148]:
```suggestion
## How to get Linkerd 2.17

The [edge-24.11.8](https://github.com/linkerd/linkerd2/releases/tag/edge-24.11.8) release is the corresponding edge release for Linkerd 2.17. See the [Linkerd releases page](/releases/) for more.

[Buoyant](https://buoyant.io/), the creators of Linkerd, has additionally released
[Buoyant Enterprise for Linkerd 2.17.0](https://buoyant.io/blog/announcing-linkerd-2-17/) and
published a
[Linkerd 2.17 changelog](https://docs.buoyant.io/release-notes/buoyant-enterprise-linkerd/enterprise-2.17.0/)
with additional guidance and content.
```

**Pull request merged**